fpga_interchange/examples/chipdb.cmake

```diff
@@ -0,0 +1,134 @@
+function(create_rapidwright_device_db)
+    set(options)
+    set(oneValueArgs device part)
+    set(multiValueArgs)
+
+    cmake_parse_arguments(
+        create_rapidwright_device_db
+        "${options}"
+        "${oneValueArgs}"
+        "${multiValueArgs}"
+        ${ARGN}
+    )
+
+    set(device ${create_rapidwright_device_db_device})
+    set(part ${create_rapidwright_device_db_part})
+    set(rapidwright_device_db ${CMAKE_CURRENT_BINARY_DIR}/${part}.device)
+    add_custom_command(
+        OUTPUT ${rapidwright_device_db}
+        COMMAND
+            RAPIDWRIGHT_PATH=${RAPIDWRIGHT_PATH}
+            ${RAPIDWRIGHT_PATH}/scripts/invoke_rapidwright.sh
```

Comment:
Could the path to `invoke_rapidwright.sh` be put in a cmake variable? The whole concept of `RAPIDWRIGHT_PATH` kind of breaks down when rapidwright is packaged instead of just lying around in a local git repo, and it would be nice to be able to configure this sinlge entry point to fit the installation method.

Reply:
I agree with this.

Reply:
Done